benchmarks/nyc_taxi/modin-ray/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# start cluster
+ray up modin-cluster.yaml -y
+
+ray submit modin-cluster.yaml scale_cluster.py
+
+# run full benchmark
+ray submit modin-cluster.yaml nyc_taxi_precipitation.py 
+
+# run two more times
+ray submit modin-cluster.yaml nyc_taxi_precipitation.py 
+ray submit modin-cluster.yaml nyc_taxi_precipitation.py 
```

Comment:
Tested this script on smaller instances / less data and it doesn't appear to be doing any caching as the io/compute times remained pretty steady.